index.js

```diff
@@ -0,0 +1,110 @@
+// import { endsWith } from 'lodash';
+// import './style.css';
+
+let todos = JSON.parse(localStorage.getItem('localStore')) || [];
+function DisplayTodos() {
+  const todoList = document.querySelector('#todo-list');
+  todoList.innerHTML = '';
+
+  todos.forEach((todo) => {
+    // Create element for rendering input
+
+    const todoItem = document.createElement('div');
+    todoItem.classList.add('todo-item');
+    const label = document.createElement('label');
+    const input = document.createElement('input');
+    const content = document.createElement('div');
+    const actions = document.createElement('div');
+    const trashIcon = document.createElement('i');
+    trashIcon.classList.add('fa-regular');
+    trashIcon.classList.add('fa-trash-can');
+
+    input.type = 'checkbox';
+    input.checked = todo.completed;
+
+    content.classList.add('todo-content');
+    actions.classList.add('actions');
+
+    content.innerHTML = `<input class="edit" type="text" value="${todo.content}" readonly>`;
+
+    label.appendChild(input);
+    actions.appendChild(trashIcon);
+    todoItem.appendChild(label);
+    todoItem.appendChild(content);
+    todoItem.appendChild(actions);
+    todoList.appendChild(todoItem);
```

Comment:
I think it would be better to try to avoid having duplicated code. Instead, you should reuse your code when possible